vidarrtest-regression.json.in

```diff
@@ -103,7 +103,7 @@
             {
                 "metrics_calculate": "@CHECKOUT@/tests/calculate.sh",
                 "metrics_compare": "@CHECKOUT@/tests/compare.sh",
-                "output_metrics": "/.mounts/labs/gsi/testdata/delly/output_metrics/UnmatchedTest.metrics",
```

Comment:
I think we should keep it as 'UnmatchedTest' - there is no OICR-specific or patient-specific info here, may be helpful when developing RTs further